Formula/pijul.rb

```diff
@@ -0,0 +1,25 @@
+class Pijul < Formula
+  desc "Patch-based distributed version control system, easy to use and fast"
+  homepage "https://pijul.org"
+  url "https://crates.io/api/v1/crates/pijul/0.10.1/download"
```

Comment:
Not sure if we want to use crates.io or a tarball from the homepage?

Reply:
Probably the one from the homepage, IMO.

Reply:
It’s not a *problem* to make this change, but note that pijul.org only provides version 0.10.**0**, not 0.10.**1**. I’ll assume I should switch to download from <https://pijul.org/releases/pijul-0.10.0.tar.gz> regardless unless I hear otherwise.

Reply:
Might be worth trying to find out what the deal is with that, whether someone forgot to update the website or whether 0.10.1 isn't considered a formal release or whatnot.